sagemaker-pipelines/tabular/lambda-step/sagemaker-pipelines-lambda-step.ipynb

```diff
@@ -0,0 +1,903 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    This Lambda function creates and Endpoint Configuration and deploys a model to an Endpoint. 
</pre><p>creates and --&gt; creates an</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Fixed